### PR TITLE
ARC-677 permissions doc update that includes write permissions for deployments

### PR DESCRIPTION
**Pull request by rachellerathbone**
PR for permissions updates that will go out before we mess around with permissions settings in prod. This PR will only go to prod in the event that we need to revert deployment permissions back to including `write`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,11 +107,37 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 
 #### GitHub permission scopes
 
+##### Repository Permissions
+
+|**Permission scope**|**Why the app needs it**|
+|---|---|
+|**Read** access to contents & metadata |**Contents** (aka code) and **metadata** are needed to sync development information to Jira. <br><br> **Medadata:** All GitHub apps have read-only metadata permission set by default. This is a [mandatory requirement by GitHub](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) and is needed to provide access to a collection of read-only endpoints with metadata for various resources. These endpoints do not provide sensitive private repository information. Read-only metadata permissions are used for the following webhook: <br> - repository (all events excluding `deleted`) <br><br> **Contents:** Read-only content permissions are used for the following webhooks: <br> - commit comment <br> - create <br> - delete <br> - push <br> - workflow run|
```
Comment (mboudreau) [inline README.md:114]: why are we using `<br>`?  We don't need HTML in markdown...

File: README.md [hunk 2]
```diff
@@ -104,11 +107,37 @@ Read, Write, and Admin for Development Information (branches, commits, and pull
 
 #### GitHub permission scopes
 
+##### Repository Permissions
+
+|**Permission scope**|**Why the app needs it**|
```
Comment (mboudreau) [inline README.md:112]: I'm not sure if this section needs a table.  I feel like it makes it harder to read when all the content is list based anyways

Comment (rachellerathbone) [reply README.md:112]: It's currently a table on main. I've just added extra content. 

Did you look at the file? I don't think it's confusing.

Screen Shot 2022-01-18 at 9 19 20 am

File: README.md (hunk 1 above)
Comment (rachellerathbone) [reply README.md:114]: Without the break tags, the content looks like this:

Screen Shot 2022-01-18 at 9 20 34 am

Vs this:

Screen Shot 2022-01-18 at 9 21 53 am

**Pull request closed without merging**